### PR TITLE
LPS-135082 Drools Upgrade tests failing on IBM DB2 11.1

### PR DESCRIPTION
**Pull request by AliciaGarciaGarcia**
I couldn't test on DB2, but on 6.2 modifying the Database It made similar error with MySQL about the companyId column.

DL upgrade are made before DDM so in some cases companyID column was not created.

I tested https://issues.liferay.com/browse/LPS-133831 and https://issues.liferay.com/browse/LPS-133766 manually and seems to be ok

### CODE REVIEW

File: modules/apps/document-library/document-library-service/src/main/java/com/liferay/document/library/internal/upgrade/DLServiceUpgrade.java [hunk 1]
```diff
@@ -110,7 +109,7 @@ public void register(Registry registry) {
 				UpgradeDLFileEntryType());
 
 		registry.register(
-			"3.2.1", "3.2.2", new UpgradeDDMStructureLinkDLFileEntryType());
```
Comment (AliciaGarciaGarcia) [inline modules/apps/document-library/document-library-service/src/main/java/com/liferay/document/library/internal/upgrade/DLServiceUpgrade.java:113]: @austinchiang I think that we can remove the version (downloading the bnd too) cause it is not published?

but I prefer to ask you before doing that,.

**Comment by AliciaGarciaGarcia** [comment]
cc// @achaparro

this it's the PR

**Comment by AliciaGarciaGarcia** [comment]
ci:test:relevant

**Comment by AliciaGarciaGarcia** [comment]
ci:test:sf

**Comment by AliciaGarciaGarcia** [comment]
@adolfopa I know that this need to be sent to Data engine team but I prefer you to review it

**Comment by AliciaGarciaGarcia** [comment]
ci:test:relevant

**Review by adolfopa — APPROVED** [review]

**Comment by AliciaGarciaGarcia** [comment]
Created : https://github.com/liferay-data-engine/liferay-portal/pull/569 to review by @liferay-data-engine

**Pull request closed without merging**